### PR TITLE
Add ORBITF435 target

### PR DESCRIPTION
**Pull request by orbittechnologyx**
The configurations for ORBITF435 target

### CODE REVIEW

File: configs/ORBITF435/config.h [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+#define GYRO_1_SPI_INSTANCE SPI1
+#define GYRO_1_ALIGN CW270_DEG
+#define GYRO_1_ALIGN_YAW 2700
+
+#define PINIO1_PIN PC8
+#define PINIO1_CONFIG 129
+#define PINIO1_BOX 40
+
+#define SERIALRX_UART SERIAL_PORT_USART2
+#define ESC_SENSOR_UART SERIAL_PORT_UART5
```
Comment (haslinghuis) [inline configs/ORBITF435/config.h:127]:
```suggestion
#define ESC_SENSOR_UART SERIAL_PORT_UART5

```

**Review by haslinghuis — APPROVED** [review]
- Using 2 GPIO ports is good.
- Using 3 timers - using two 4-channel timers for 8 motors for when BITBANG is disabled would be better.

```
2 PB0   TMR 3_CH3
2 PB1   TMR 3_CH4
1 PB10 TMR 2_CH3
1 PB11 TMR 2_CH4
1 PA15 TMR 2_CH1
1 PB6   TMR 4_CH1
1 PB7   TMR 4_CH2
1 PA1   TMR 2_CH2
```
```
M1: 2 PB0   (2)TMR 3_CH3
M2: 2 PB1   (2)TMR 3_CH4

M5: 1 PA15  (1)TMR 2_CH1
M8: 1 PA1   (1)TMR 2_CH2
M3: 1 PB10  (1)TMR 2_CH3
M4: 1 PB11  (1)TMR 2_CH4

M6: 1 PB6   (1)TMR 4_CH1
M7: 1 PB7   (1)TMR 4_CH2
```

File: Manufacturers.md [hunk 2]
```diff
@@ -85,6 +85,7 @@ This is the official list of manufacturer ids (`manufacturer_id` in the target c
 |NGUA|NG.UAVP|https://ng.uavp.ch/Shop|
 |NYXT|Nyxtronics|https://nyxtronics.com|
 |ODTY|OddityRC|http://www.oddityrc.com|
+|ORBT|Orbit Technology|https://www.orbitteknoloji.com.tr/|
 |OPEN|OpenPilot|https://librepilot.atlassian.net/wiki/spaces/LPDOC/pages/31588369/Supported+Hardware|
 |ORQA|Orqa Ltd|https://orqafpv.com/|
```
Comment (ot0tot) [inline Manufacturers.md:90]:
```suggestion
|OPEN|OpenPilot|https://librepilot.atlassian.net/wiki/spaces/LPDOC/pages/31588369/Supported+Hardware|
|ORBT|Orbit Technology|https://www.orbitteknoloji.com.tr/|
|ORQA|Orqa Ltd|https://orqafpv.com/|
```

File: configs/ORBITF435/config.h [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    TIMER_PIN_MAP( 1, MOTOR2_PIN, 2,  0) \
+    TIMER_PIN_MAP( 2, MOTOR3_PIN, 1,  0) \
+    TIMER_PIN_MAP( 3, MOTOR4_PIN, 1,  0) \
+    TIMER_PIN_MAP( 4, MOTOR5_PIN, 1,  0) \
+    TIMER_PIN_MAP( 5, MOTOR6_PIN, 1,  0) \
+    TIMER_PIN_MAP( 6, MOTOR7_PIN, 1,  0) \
+    TIMER_PIN_MAP( 7, MOTOR8_PIN, 1,  0) \
+    TIMER_PIN_MAP( 8, LED_STRIP_PIN,  2,  0) \
+    TIMER_PIN_MAP( 9, CAMERA_CONTROL_PIN, 3, -1) \
+    TIMER_PIN_MAP( 10, RX_PPM_PIN, 2, -1)
```
Comment (ot0tot) [inline configs/ORBITF435/config.h:97]: TIM2 and TIM5 are already used by motors and LED_STRIP, respectively. 
```suggestion
    TIMER_PIN_MAP( 10, RX_PPM_PIN, 3, -1) 
```

File: configs/ORBITF435/config.h [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+#define MAG_I2C_INSTANCE (I2CDEV_1)
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+#define FLASH_SPI_INSTANCE SPI3
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+#define DEFAULT_DSHOT_BITBANG DSHOT_BITBANG_ON
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SCALE 101 //or it may 102
+#define DEFAULT_CURRENT_METER_SCALE 400
```
Comment (ot0tot) [inline configs/ORBITF435/config.h:109]: Does this FC come with an ESC? If not, the default current meter scale should not be set.

Comment (orbittechnologyx) [reply configs/ORBITF435/config.h]: Currently doesn't come but in future it will come with an ESC. So I removed  current meter scale value for now.

**Review by haslinghuis — APPROVED** [review]

**Review by ot0tot — APPROVED** [review]

**Comment by orbittechnologyx** [comment]
Hi @haslinghuis @ot0tot 
We decided to replace esc telemetry from uart5 to uart3. I updated the config.h file for it. In this way, I think we can prevent hijacking at least a little bit. Do you think it's ok?

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**